sdk/communication/Azure.Communication.NetworkTraversal/src/CommunicationRelayClient.cs

```diff
@@ -93,15 +93,16 @@ protected CommunicationRelayClient()
         /// <summary>Gets a Relay Configuration for a <see cref="CommunicationUserIdentifier"/>.</summary>
         /// <param name="communicationUser">The <see cref="CommunicationUserIdentifier"/> for whom to issue a token.</param>
         /// <param name="routeType"> The specified <see cref="RouteType"/> for the relay request </param>
+        /// <param name="ttl"> The specified Time-to-live for the relay request </param>
         /// <param name="cancellationToken">The cancellation token to use.</param>
         /// <exception cref="RequestFailedException">The server returned an error.</exception>
-        public virtual Response<CommunicationRelayConfiguration> GetRelayConfiguration(CommunicationUserIdentifier communicationUser = null, RouteType? routeType = null, CancellationToken cancellationToken = default)
+        public virtual Response<CommunicationRelayConfiguration> GetRelayConfiguration(CommunicationUserIdentifier communicationUser = null, RouteType? routeType = null, int? ttl = null, CancellationToken cancellationToken = default)
```

Comment:
is `ttl` a common acronym that customers would know? We normally don't use acronyms in our SDKs. Suggest changing to timeToLive. 

It will also be helpful to specify the unit of time in the summary description. (which is seconds) because that is not immediately obvious to me.